docs/guides/Makefile

```
@@ -20,3 +20,11 @@ clean:
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
 	$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+update_redirects:
+	# If files got moved there should be a redirect in redirects.txt
```

```suggestion
	# This updates redirects.txt, creating redirects for any files that have moved (relative to master).
```
I tried to make it clearer what this target does.